src/styles/blocks/main.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  --column-count: 4;
+  
+  display: grid;
+  gap: 46px 48px;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  margin-block: 50px 72px;
```

Comment:
The shorthand property `margin-block` is not fully supported by all browsers yet. It's safer to use `margin-top` and `margin-bottom` for better browser compatibility.